coding/browser-support.md

```diff
@@ -0,0 +1,36 @@
+# Browser support
+
+Our baseline browser support for our websites. There are two levels of support. 
+
+## Full support
+
+Any browser with full support needs to be pixel perfect and all functionality 
+should work.
+
+| Browser | Version |
+| ------- | ------- |
+| Chrome | 55 or greater |
+| Firefox | 52 or greater |
+| Safari | 11 or greater |
+| IE | 11 or greater |
+| Edge | 15 or greater |
+| Opera | 48 or greater |
+
+## Accessible support
+
+The content on these browsers should be readable but my not look perfect.
+
+| Browser | Version |
+| ------- | ------- |
+| Chrome | 49 |
+| Firefox | 45 |
+| Edge | 14 |
+
+Any browser not mentioned here is not supported and therefore not tested.
+
+### Testing on different browsers
+
+The team has an account with [BrowserStack](https://www.browserstack.com). To
```

Comment:
`The team has an account with [BrowserStack](https://www.browserstack.com) to test your site on browsers you don't have installed or different versions. You can find the teams email and password in our credentials document.`